### ISSUE TITLE
Add /skip command

### ISSUE THREAD

**Issue by tjhubz** [WarzoneMC/Warzone#72]
Allows you to force skip the next map

Syntax: /skip (number of maps you want to skip) or /skip will just skip one map.

**Comment by DeternoMC** [comment]
pretty sure /cycle does this besides a certain number of maps.

**Comment by BennyDoesTheStuff** [comment]
@DeternoMC 
Nope, using /cycle just cycles the current map. Using /cycle [seconds] is how long unitl the current map will be cycled.

**Comment by JapaneseSquid** [comment]
I don’t think this is needed. You can just do /cycle 1 and it’ll have the same effect.

**Comment by MatrixTunnel** [comment]
That only cycles the map. Skipping 7 maps would take a while since you would have to load each match then cycle again
